packages/ember-metal/lib/is_empty.js

```diff
@@ -39,7 +40,7 @@ export default function isEmpty(obj) {
 
   let objectType = typeof obj;
 
-  if (objectType === 'object') {
+  if (objectType instanceof EmberMap) {
```

Comment:
not sure if this matters, but this would not work if 2 embers where present on the same page. As there would be two different `EmberMap` and the `isEmpty` of one, could not work with the other.